pkg/databackup/constants.go

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package databackup
 
 const (
+	DATABACKUP_LOCK_NAME = "DataBackup"
```

Comment:
Naming style for constants: Go follows the camel case naming convention for constants, so I suggest renaming DATABACKUP_LOCK_NAME to DatabackupLockName.

Reply:
fixed.